.github/workflows/test.yaml

```diff
@@ -10,13 +10,15 @@ jobs:
   test:
     name: test
     strategy:
+      fail-fast: false
       matrix:
-        pyver: [3.8]
+        # 3.9 is omitted since there's no stackvana build for py3.9
+        pyver: ["3.8", "3.10"]
```

Comment:
For time being, I want to test on earlier versions of the stack which I setup and ran. Once we get the CI established more rigorously, we can drop py3.8